app/formSchema/uiSchema.ts

```diff
@@ -575,7 +577,7 @@ const uiSchema = {
     'ui:options': {
       maxLength: MAX_LONG_INPUT_LENGTH,
       fileTypes:
-        '.eml, .msg, .txt, .pdf, .doc, .docx, .xlsx, .xlsx, .xml, .jpg, .jpeg, .png',
+        '.eml, .msg, .txt, .pdf, .doc, .docx, .xls, .xlsx, .xml, .jpg, .jpeg, .png',
```

Comment:
```suggestion
        `.eml, .msg, .txt, .pdf, .doc, .docx, .xml, .jpg, .jpeg, .png, ${EXCEL_FILE_EXTENSIONS}`,
```

Reply:
I dunno I considered that but it seems messy and less readable.

Reply:
It's a bit more verbose but it ensures that you're consistent. Ideally we'd have all of these extensions in named constants and we'd just join all of them. Some of the extensions listed in this file are not necessarily well known (e.g. `.mpp`) and should really be in a constant.